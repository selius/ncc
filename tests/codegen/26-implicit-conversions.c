int main()
{
	int a;
	float b;

	b = 3.645;
	a = b;
	__print_int(a);

	b = a;
	__print_float(b);

	a = 975;
	b = a;
	__print_float(b);

	a = b;
	__print_int(a);

	return 0;
}
