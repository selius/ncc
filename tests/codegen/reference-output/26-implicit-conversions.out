3
3.000000
975.000000
975
